### PR TITLE
[Support/BLAKE3] Remove some dead stores

### PR DESCRIPTION
**Pull request by tstellar**
These were caught by the clang static analyzer.

### CODE REVIEW

**Comment by akyrtzi** [comment]
I'd like that we keep the changes with upstream BLAKE3 to a minimum, could you open a PR against https://github.com/BLAKE3-team/BLAKE3? Don't merge this until the BLAKE3 upstream PR is accepted and merged.

**Comment by tstellar** [comment]
PR for upstream: https://github.com/BLAKE3-team/BLAKE3/pull/398

**Review by akyrtzi — APPROVED** [review]

**Pull request merged**
